File: Python/tests/test_dd_hyperbolic.py
import pytest

from hbayesdm.models import dd_hyperbolic


def test_dd_hyperbolic():
    _ = dd_hyperbolic(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_dd_hyperbolic.R
context("Test dd_hyperbolic")
library(hBayesDM)

test_that("Test dd_hyperbolic", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(dd_hyperbolic(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
